repo/maintenancestats/types.go

```diff
@@ -0,0 +1,449 @@
+package maintenancestats
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/kopia/kopia/internal/contentlog"
+)
+
+// RawStats holds the raw data for maintenance statistics
+type RawStats struct {
+	Kind string          `json:"kind,omitempty"`
+	Raw  json.RawMessage `json:"raw,omitempty"`
```

Comment:
rename this struct to `Extra` and the field below to `Data`

```suggestion
type Extra struct {
	Kind string          `json:"kind,omitempty"`
	Data json.RawMessage `json:"data,omitempty"`
```